pkg/gateway/nginx.go

```diff
@@ -143,12 +151,18 @@ func renderConfig(cfg *NGINXConfig, sm *ServiceMap) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func newLoggingNGINXManager() NGINXManager {
-	return &loggingNGINXManager{status: nginxStatusStopped}
+func newLoggingNGINXManager(hp int) NGINXManager {
+	manager := loggingNGINXManager{
+		status: nginxStatusStopped,
+		cfg:    newNGINXConfig(hp),
+	}
+	log.Printf("created newLoggingNGINXManager: %+v", manager)
+	return &manager
 }
 
 type loggingNGINXManager struct {
 	status string
+	cfg    NGINXConfig
```

Comment:
Yes. The value that you're proposing we get by passing the config in is attainable in both cases, not just the loggingNGINXManager case. I don't see why we would pass the config in, which is only intended for the live nginxManager right now. If loggingNGINXManager needs a config in the future, we will solve it then.